lib/core/src/animation/differential_time_animator.rs

```diff
@@ -0,0 +1,47 @@
+use super::ContinuousTimeAnimator;
+use super::FnAnimation;
+
+
+
+// ====================================
+// === DifferentialTimeAnimatorData ===
+// ====================================
+
+struct DifferentialTimeAnimatorData {
```

Comment:
I still think the name is misleading. Citing: `In mathematics, differential refers to infinitesimal differences or to the derivatives of functions`, where `In mathematics, infinitesimals are things so small that there is no way to measure them.` We can measure the differences here, so the naming is confusing to me.